pkg/config/setup/config.go

```diff
@@ -318,7 +318,7 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("network_check.use_core_loader", false)
 
 	// If true, then the go loader will be prioritized over the python loader.
-	config.BindEnvAndSetDefault("prioritize_go_check_loader", false)
+	config.BindEnvAndSetDefault("prioritize_go_check_loader", true)
```

Comment:
won't this set the priority to use the IoT go disk / network checks over the python ones? I'm not sure we can do this yet as those lack a lot of features we're in the process of porting over with the v2 checks

Reply:
No because those two checks have a configuration to determine whether to run the Go or Python version, and it default to Python for now, see https://github.com/DataDog/datadog-agent/pull/36526 for details.

That's what I'm referring to here:
> The SNMP check had a special handling to try loading with Go first then Python, so the behavior is the same.
Each of the other checks has a configuration to pick whether to run the Go or the Python version, so there should be no impact on that side (it doesn't depend on loader priority).

Also for the record I tested this change locally and on internal k8s deployments and we're still running the Python version for both checks.

Reply:
Yep I think so